app/views/shared/proposals/_tags_form.html.haml

```diff
@@ -1,13 +1,12 @@
+%h3 Review Tags
```

Comment:
Remove the public_tags? check in this file since you'll check it before calling this partial.